src/main/java/net/smartcosmos/dto/metadata/MetadataCreate.java

```diff
@@ -0,0 +1,37 @@
+package net.smartcosmos.dto.metadata;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Setter;
+
+import java.beans.ConstructorProperties;
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+@JsonIgnoreProperties(value = {"version"}, ignoreUnknown = true)
+public class MetadataCreate {
+    private static final int VERSION = 1;
```

Comment:
I applaud the thought but naming the version constant `VERSION` is a little counter-productive.  It should be something like `VERSION_1` or not present at all.